UPGRADE.md

```diff
@@ -2,7 +2,17 @@
 
 ## dev-develop
 
-## Dependency updates
+### Deprecations
+
+Removed following Classes:
+
+* Sulu\Bundle\CoreBundle\Build\CacheBuilder
```

Comment:
Shouldn't the `CacheBuilder` work as it does currently? Clear the cache if possible with the command? Or do we only support Symfony 3.4 after this PR?

Reply:
as said before the depraction notice says that the class will be removed with 2.0